addr/sorting.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	ma "github.com/multiformats/go-multiaddr"
 	manet "github.com/multiformats/go-multiaddr-net"
-	mafmt "github.com/whyrusleeping/mafmt"
+	"github.com/whyrusleeping/mafmt"
```

Comment:
another lost qualification

Reply:
I think this case is different than the one @lanzafame pointed out (https://github.com/libp2p/go-libp2p-peerstore/pull/30#discussion_r214561252). Here the imported namespace matches the import path. In the other, it didn't:

```
package manet
```

from https://github.com/multiformats/go-multiaddr-net/blob/master/net.go

Reply:
@raulk I suggest keeping it for consistency.

Reply:
@raulk ping^

Reply:
Sorry, missed this. Will re-add the import alias before the merge, and will update my linter settings to not remove redundant import aliases.